x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -1680,7 +1680,6 @@
     "discover.bucketIntervalTooltip.tooLargeBucketsText": "大きすぎるバケット",
     "discover.bucketIntervalTooltip.tooManyBucketsText": "バケットが多すぎます",
     "discover.clearSelection": "選択した項目をクリア",
-    "discover.context.breadcrumb": "{indexPatternTitle}#{docId} のコンテキスト",
```

Comment:
Whether this translations needs to be updated manually or not?

Reply:
unless you're good in asian languages, you don't need to update this manually.